README.md

```diff
@@ -1,21 +1,23 @@
-<h2>Overview</h2>
+<h1>Splunk APM lambda layers</h1>
```

Comment:
Not terribly important, but maybe next time we can favor markdown `#` and `##` etc for making headers, instead of embedding html?

Reply:
That's a good idea. I don't even know why it's HTML in the first place ;)